example/quantization/common

```diff
@@ -0,0 +1 @@
+../image-classification/common
```

Comment:
Is this a sym link? Does it work on windows?

Reply:
Has this been addressed?